### PR TITLE
Update next/prev adjacency + tests to match engine

### PR DESCRIPTION
**Pull request by mlfarrell**
Not to merge until after https://github.com/KittyCAD/engine/pull/2518 is merged
obv these tests wont pass until then either

once this merges, so should https://github.com/KittyCAD/kcl-samples/pull/64

### CODE REVIEW

**Comment by lf94** [comment]
Just so I understand this is a pretty big breaking change right? It seems to invert everything?

**Comment by mlfarrell** [comment]
@lf94  Its a backwards breaking change yes.  but its asked for specifically by @josh and @JordanNoone for consistency

**Review by jessfraz — APPROVED** [review]

File: src/wasm-lib/tests/executor/main.rs [hunk 1]
```diff
@@ -1328,11 +1328,7 @@ const bracket = startSketchOn('XY')
 "#;
 
     let result = execute_and_snapshot(code, UnitLength::Mm).await;
-    assert!(result.is_err());
-    assert_eq!(
-        result.err().unwrap().to_string(),
-        r#"engine: KclErrorDetails { source_ranges: [SourceRange([1329, 1430])], message: "Modeling command failed: [ApiError { error_code: BadRequest, message: \"Fillet failed\" }]" }"#
-    );
+    assert!(!result.is_err());
```
Comment (jtran) [inline src/wasm-lib/tests/executor/main.rs:1331]: This is giving me a lint warning.

```suggestion
    assert!(result.is_ok());
```

Comment (mlfarrell) [reply src/wasm-lib/tests/executor/main.rs:1331]: yep, already handled it locally.  i'm backed up on deployment of engine (separate fix needed for a test here) before I push

**Comment by mlfarrell** [comment]
waiting on https://github.com/KittyCAD/engine/actions/runs/10688775816, then will push and all tests should pass

**Pull request merged**